config/initializers/hypershield.rb

```diff
@@ -0,0 +1,18 @@
+Hypershield.schemas = {
+  hypershield: {
+    hide: %w[
+      auth_data_dump
```

Comment:
should we qualify some of the various fields we are hiding?

The README shows you can write "table_name.field":

```ruby
Hypershield.schemas = {
  hypershield: {
    hide: %w(encrypted password token secret),
    show: %w(ahoy_visits.visit_token)
  }
}
```

I'm thinking about this in case there are fields called "X" that should be hidden in a table but not for another :D

Reply:
I kinda like the idea of keeping these "not table specific" since that will cover new tables with these fields if they are ever added and there is nothing we have to do or worry about in that case.